lib/core/elb_wrapper.rb

```diff
@@ -11,8 +11,13 @@ def elb_client
   
   public
   
-  def elb_arn(region, account_id, elb_name)
-    "arn:aws:elasticloadbalancing:#{region}:#{account_id}:loadbalancer/#{elb_name}"
+  def elb_arn(elb_name)
+    account_id = Aws::IAM::CurrentUser.new.arn.match(/^arn:aws:iam::(\d+)/)[1]
+    "arn:aws:elasticloadbalancing:#{availability_zone}:#{account_id}:loadbalancer/#{elb_name}"
+  end
```

Comment:
(Talked in person: The user ARN has the account ID, which we use for building another ARN. There could be a better way.)